profile/profile.yaml

```diff
@@ -0,0 +1,99 @@
+# Preferred name (Characters only)
+name: "Bhavya Vinod"
+# Your GitHub repo root URL
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_BhavyaVinod
+
+# Introduce yourself, your academic and professional background.
+# What excites you about the software industry and your passion. At least 80 words
+summary: I am a results-oriented person who is passionate about software development. I have more than two years of experience as an IT professional. This experience helped me to build a detail-oriented, analytical mindset and exposed me to the work culture and the agile methodologies followed in the large-scale industries. Before pursuing a career in Data Science, I completed my degree in Data analytics for business. This program covered Machine learning, Data visualization, Financial Analytics, and Advanced statistics. Through my passion to understand the needs of clients and the ability to quickly learn new technologies, I will be a great fit to the team. 
+
+# Group your skills by level.
+# Your skills can be anything software related (e.g. language, frameworks, methodologies, software, etc.).
+skills:
+  # At least 5 skills
+  proficient:
+    - Java
+    - Linux/Bash
+    - RDBMS/SQL
+    - Agile/Scrum
+    - Git
+  # At least 5 skills
+  competent:
+    - Python
+    - Tableau
+    - Numpy
+    - Pandas
+    - R
+  # At least 5 skills
+  familiar:
+    - AWS
+    - Docker
+    - Computer Networking
+    - Google Cloud Platform
+    - Power BI
+
+# List of Jarvis projects
+jarvis_projects:
+  - name: "Cluster Monitor"
+    git_url: "https://github.com/jarviscanada/jarvis_data_eng_BhavyaVinod/tree/develop/linux_sql"
+    # The project description allows hiring managers to understand your project and technologies you used(e.g. programming languages, libraries, algorithms, hardware, tools, etc.).
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: Implemented a Linux/ SQL cluster monitoring solution to collect hardware specifications of each nodes and monitors resource usage such as CPU and memory utilization in real time. The collected data will be stored in a PostgreSQL server that runs inside a docker.
```

Comment:
description: Implemented a Linux/ SQL cluster monitoring solution to collect hardware specifications of each node and monitor resource usage such as CPU and memory utilization in real-time. The collected data will be stored in a PostgreSQL server that runs inside a docker.